app/views/token-based-resume/save_return_resend_link.html.erb

```diff
@@ -6,15 +6,18 @@
     <% else %>
       <h1 class="govuk-heading-l">Request a secure link to your application</h1>
     <% end %>
+   <%= form_for @application, url: "/sponsor-a-child/resend-link", method: :post do |f| %>
+
+    <%= f.govuk_error_summary link_base_errors_to: :email %>
+    <% end %>
 
     <p class="govuk-body">
         Confirm your email address and we will send you a new one so you can continue your application.
     </p>
 
     <%= form_for @application, url: "/sponsor-a-child/resend-link", method: :post do |f| %>
       <%= f.govuk_text_field :email,
-                             label: { text: t('email.full', scope: "unaccompanied_minor.questions"), size: 'm', hidden: true },
-                             hint: { text: t('email.hint', scope: "unaccompanied_minor.questions")},
+                             label: { text: t('email.label_email', scope: "unaccompanied_minor.questions") },
```

Comment:
Keep old hint text as well as a label, should to tell user to use an email they can access